tectonic/xetexini.c

```diff
@@ -5,6 +5,414 @@
 #include <tectonic/core-bridge.h>
 #include <tectonic/dpx-pdfobj.h> /* pdf_files_{init,close} */
 
+memory_word *the_eqtb;
```

Comment:
Can you add a comment here saying that these variables are prototyped in xetexd.h, and an analogous one in xetexd.h saying that the variables are actuallye declared in xetexini.c?